### ISSUE TITLE
Android 2.3.6 PhoneGap Integration

### ISSUE THREAD

**Issue by pamboo80** [MrSwitch/hello.js#180]
Hi,

First, a big thanks for this library. It made my life very easy. I m using hellojs on both my website and mobile app (using phonegap). Everything works fine on my Android mobile versions 4.1.2 (Jelly Bean) and 4.4.2 (KitKat) . But, when I run the same APK on the Android 2.3.6 , it fails at line 132:25 on hello.all.js  file , in the function "use" and the statement "var self = this.utils.objectCreate(this);"  . The error message is "TypeError:Object prototype may only be an Object or null at Object.create[as objectCreate] (native) at Function.use". This could be an issue with the version of web-kit used in 2.3.6 as it works fine in newer versions of Android phones. When I alert 'this' value , it is not null and it is same value as displayed in working versions of Android and the 2.3.6version. But , don't know why it says "TypeError". Can you help me on this?

http://jimbergman.net/webkit-version-in-android-version/
http://kangax.github.io/compat-table/es5/#webkit

Please , give some insight on this so that if I can fix it to run on Android 2.3.6 version too.

**Comment by MrSwitch** [comment]
Hi @pamboo80 

Do the tests pass http://adodson.com/hello.js/tests/specs/ in Android 2.3.6 browser?

**Comment by pamboo80** [comment]
No, Passes:87 failures:63 100%

Failure on utils/clone 
should not clone Blob values

hello.api
*should assign a complete event

hello.use
hello.api 
...

**Comment by MrSwitch** [comment]
@pamboo80 your going to have to help us all on this. Try running the function utils.objectCreate outside of HelloJS perhaps that should give us some insight into what about Object.create it doesn't like

**Comment by MrSwitch** [comment]
Maybe it doesn't like being reassigned, does this trigger an alert with "[object Object]"

```javascript
var a = Object.create;
alert(a({}));
```

**Comment by pamboo80** [comment]
Sure. Few mins please. Let me check and update here. Thanks.

**Comment by pamboo80** [comment]
Yes. [object Object] . Also when I called hello.utils.objectCreate(null) ; no error was there.

**Comment by MrSwitch** [comment]
I have no further suggestions, sorry.

**Comment by pamboo80** [comment]
No issues. If I find any solutions , let me add here. Thanks.
